core/wasm/runtest/generate-wasm/import-memory/src/main.rs

```diff
@@ -42,15 +43,31 @@ fn externalize_mem(
     builder.build()
 }
 
+fn help() {
```

Comment:
Why do you need to make this a function

Reply:
Copy pasta, but actually I think makes sense to extract conversion code well.